src/main/java/com/dev/cinema/service/mapper/CinemaHallMapper.java

```diff
@@ -0,0 +1,24 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.dto.request.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.response.CinemaHallResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapper {
+    public CinemaHallResponseDto toDto(CinemaHall cinemaHall) {
+        CinemaHallResponseDto cinemaHallResponseDto = new CinemaHallResponseDto();
+        cinemaHallResponseDto.setId(cinemaHall.getId());
+        cinemaHallResponseDto.setCapacity(cinemaHall.getCapacity());
+        cinemaHallResponseDto.setDescription(cinemaHallResponseDto.getDescription());
```

Comment:
is everything ok on this line?

Reply:
was inattentive. changed getting description from Cinema Hall object.